axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/PrimaryEventStore.java

```diff
@@ -423,7 +423,7 @@ private WritePosition claim(int eventBlockSize, int nrOfEvents) {
         do {
             writePosition = writePositionRef.getAndAccumulate(
                     new WritePosition(nrOfEvents, totalSize),
```

Comment:
maybe it makes more sense to do something like:
`writePosition = writePositionRef.getAndUpdate(prev -> prev.incrementedWith(nrOfEvents, totalSize));`